libs/hire-requests/feature/src/lib/job-summary-modal/job-summary-modal.component.scss

```diff
@@ -0,0 +1,346 @@
+    margin-top: 0px; 
+}
+
+.parent-name {
+    font-size: 30px;
+    padding-top: 2%;
+    margin-bottom: 2%;
+    color: black;
+    text-align: center;
+    font-family: Roboto;
+    margin-top: 0px;
+}
+
+.profile-name {
+    font-size: 2.4vw;
+    padding-top: 2%;
+    margin-bottom: 2%;
+    color: white;
+    text-align: center;
+    font-family: Roboto;
+    margin-top: 0px;
+}
+
+.children-name{
+    font-size: 1.8vw;
+    padding-top: 2%;
+    margin-bottom: 2%;
+    color: black;
+    text-align: center;
+    font-family: Roboto;
+    margin-top: 0px;
+}
+
+.parent-profile-div{
+    width: 100%;
+    margin-left: auto;
+    margin-right: auto;
+    background-color: #69A88DB2;
+    border-radius: 16px;
+}
+
+.profile-picture-div{
+    width: 33.3%;
+    height: 100%;
+    float: left;
+}
+
+.details-div{
+    width: 60%;
+    height: 100%;
+    margin-left: auto;
+    margin-right: auto;
+    padding-bottom: 25px;
+}
+
+.empty-div{
+    width: 33.3%;
+    height: 100%;
+    float: right;
+}
+
+.profile-picture-wrapper{
+    width: 166px;
+    height: 166px;
+    background-color: rgba($color: #032927BF, $alpha: 0.75);
+    border-radius: 50%;
+    margin: auto;
+    display: flex;
+    align-items: center;
+    text-align: center;
+    justify-content: center;
+    margin-top: 20px;
+    margin-bottom: 25px;
+}
+
+.profile-picture {
+    width: 140px;
+    height: 140px;
+    border-radius: 50%;
+}
+
+.parent-info-small{
+    height: 6%;
+    background-color: var(--ion-background-color);
+    display:flex;
+    font-family: Roboto;
+    align-items: center;
+    color: black;
+    margin-bottom: 12px;
+    box-shadow: 1px 2px 5px rgba(0, 0, 0, 0.13);
+    border-radius: 6px;
+}
+
+.parent-info-large{
+    height: 26%;
+    background-color: var(--ion-background-color);
+    display:flex;
+    margin-top: 10px;
+    font-family: Roboto;
+    color: black;
+    margin-bottom: 12px;
+    box-shadow: 1px 2px 5px rgba(0, 0, 0, 0.13);
+    border-radius: 6px;
+}
+
+.parent-text{
+    margin-left: 12px;
+    font-size: 100%;
+    width: 35%;
+    margin: 10px;
+}
+
+.parent-text-info{
+    color: var(--ion-color-dark);
+    font-size: 100%;
+    margin-right: auto;
+    line-height: 170%;
+    margin-top: auto;
+    margin-bottom: auto;
+}
+
+.parent-text-info2{
+    color: var(--ion-color-dark);
+    font-size: 100%;
+    margin-right: auto;
+    line-height: 170%;
+    margin-top: auto;
+    margin-bottom: auto;
+}
+
+.terms-container{
+    margin-left: 20px;
+    margin-right: 20px;
+    color: black;
+    font-size: 15px;
+}
+
+.action-buttons{
+    display: flex;
+}
+
+.accept-button{
+    padding: 10px 22px;
+    gap: 10px;
+    border: 2px solid var(--ion-background-color);
+    border-radius: 10px;
+    background-color: var(--ion-color-primary);
+    margin: auto;
+    display: block;
+    margin-bottom: 30px;
+}
+
+.reject-button{
+    padding: 10px 22px;
+    gap: 10px;
+    border: 2px solid var(--ion-background-color);
```

Comment:
Is this border meant to blend in with the background? And can we maybe look at changing the reject button colour from danger just so its not necessarily "bad" to reject. Maybe we make it grey